book/src/dev/rfcs/0011-async-rust-in-zebra.md

```diff
@@ -622,18 +622,30 @@ makes code hard to read and maintain. Map the `Either` to a custom enum.
 [using-atomics]: #using-atomics
 
 If you're considering using atomics, prefer:
```

Comment:
```suggestion
If you're considering using atomics in production code, prefer:
```

Reply:
It's still pretty easy to make mistakes with atomics in test code, so I really don't want a blanket exception for tests.

For example, @jvff's initial atomics code used a weak memory ordering, which could have caused issues in future revisions. That could have been avoided by using a watch channel.

So it's still accurate for the RFC to say that watch channel are easier to use correctly.

Also the tests you've both written are covered by the next point:
> 2. using the strongest memory ordering `SeqCst`